### ISSUE TITLE
Doesn't work, it loops in login page.

### ISSUE THREAD

**Issue by drumset** [go2ismail/DotnetDesk#5]
Great work, but even if I have authenticated user, the application loops in login page. Page not found if i try to go /Config/Index Action.
How to fix?

Thanks in advance.

**Comment by Bezaeel** [comment]
Did you click the link to confirm email?

**Comment by drumset** [comment]
Yes, I followed all the registration steps but the return url is still login.
http://localhost:62672/Account/Login?ReturnUrl=%2F
The return url seems empty.

**Comment by Bezaeel** [comment]
Hi, the return URL would be empty because you're not coming from any prior directory/URL within the application, and empty return URL would lead to the index page
I have tried to reproduce the issue from registration step to logging in and it all seems fine;
Here is what i think the problem is 

- your email confirmation was not successful for this account; try with another account

- Assuming you entered the right credentials though

**Comment by alexrichi** [comment]
Hi! Indeed, after logging in Google Chrome returns the value "/", Opera returns "/ Config / Index" and then everything works.

**Comment by strenkomendencha** [comment]
As a workaround you can register and modify EmailConfirmed property of a user directly in DB
